sys/net/include/aodvv2/aodvv2.h

```diff
@@ -34,7 +35,7 @@ extern "C" {
 /**
  * @brief   Initialize the AODVv2 routing protocol.
  */
-void aodv_init(void);
+void aodv_init(kernel_pid_t interface);
```

Comment:
I'd say AODVv2 can currently only serve one interface and put that into the documentation? Multiple interfaces would be nifty, but I have a huge backlog of draft changes that I'd like to implement first...